tests/unit/components/sl-radio-group-test.js

```diff
@@ -145,3 +145,50 @@ test( "Inline false sets sl-radio children's inline property to false", function
         'Rendered component has three default (non-inline) radio buttons'
     );
 });
+
+test( 'change listener is added and removed from the correct namespace', function( assert ) {
+    const component = this.subject({
+        name: 'test',
+        template: Ember.Handlebars.compile(
+            '{{sl-radio label="One" value="one"}}'
+        )
+    });
+    const inputElement = this.$( 'input[name=test]:radio' );
+    const jQueryData = Ember.get( Ember.$, '_data' );
+    const eventData = Ember.get( jQueryData( inputElement[0], 'events' ), 'change' );
```

Comment:
Same comment left on similar logic in sl-date-picker-test.js file applies here as well.